tests/API/XCCDF/unittests/test_fix_bash_header.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
```

Comment:
this file suggests we are testing bash headers but we are also testing ansible headers, I suggest we rename it or split it into 2 files